src/renderer.js

```diff
@@ -12,10 +12,11 @@ import Repos from './components/Repos';
 import Repo from './components/Repo';
 
 const store = createStore(rootReducer);
-const { ipcRenderer } = require('electron');
+const electron = require('electron');
+const ipc = electron.ipcRenderer;
 
-ipcRenderer.on('retrievedGithubData', () => {
-  console.log('hey');
+ipc.on('retrievedGithubData', (data) => {
```

Comment:
@Alex-Tideman 
we're having trouble getting this ipcRenderer to trigger. Any chance you can spot why we can't get this to work? We're sending via WebContents in the main.js file.